### ISSUE TITLE
Playwright reporter not reporting test cases

### ISSUE THREAD

**Issue by alkanna** [qase-tms/qase-javascript#91]
I get the following notice at the end of a test run :
```
qase: Test should log out properly (Qase ID: 11) passed
qase: Start publishing: should log out properly (Qase ID: 11)


  1 passed (2s)
qase: No testcases were matched. Ensure that your tests are declared correctly.
No testcases were matched. Ensure that your tests are declared correctly
```

This case with ID 11 exists, I tried with other existing as well but it does not seem to work.
The project code is right as well.

Is there some additional verbose mode maybe to help troubleshoot this @alexneo2003 ?

**Comment by alkanna** [comment]
It looks like with two test cases instead of one in playwright, one successful and one failing, the successful one is indeed reported correctly. However, the failing one is not shown at all in Qase.

**Comment by alkanna** [comment]
I intercepted http traffic between the reporter and qase, I found out that the request gets aborted before a response is received :
<img width="594" alt="CleanShot 2021-11-30 at 14 48 38@2x" src="https://user-images.githubusercontent.com/30343764/144059167-832c2e5b-6535-41fb-8b76-f012e3a24c80.png">

Additionally, when I re ran the same request manually using cUrl, the failed case was properly appended to the same run.

**Comment by alkanna** [comment]
This time I ran my two tests again in playwright debug mode, and as it probably exits later than in headless mode, it worked properly :
![CleanShot 2021-11-30 at 14 53 54](https://user-images.githubusercontent.com/30343764/144059991-186b8eca-f658-431f-aac5-685a2e87f68b.png)

**Comment by kamilsk** [comment]
fixed and published as `playwright-qase-reporter@1.1.2`, thanks a lot @alexneo2003
